crates/holochain/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+- Prevent duplicate calls to `init`. Previously, calling `init` directly would result in 2 calls to `init` if the zome had
```

Comment:
What's going on here? Something happened in a rebase or did I miss including this?

Reply:
Yeah I think it was a merge mistake. Fixed